package.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "hydraclient.js",
+    "version": "0.0.1",
+    "scripts": {
+        "test": "node node_modules/karma/bin/karma start",
+        "test-once": "node node_modules/karma/bin/karma start --single-run"
```

Comment:
Changed to 'test' (single run) and 'live-test' (for continuous running test session)